lib/formatters/visualstudio.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @fileoverview VisualStudio reporter
+ * @author Ronald Pijnacker
+ */
```

Comment:
Please add copyright and license information. See: http://eslint.org/docs/1.10.3/developer-guide/code-conventions.html#file-format